tsl/src/planner.c

```diff
@@ -315,12 +315,47 @@ tsl_create_distributed_insert_path(PlannerInfo *root, ModifyTablePath *mtpath, I
 					if (rte->rtekind == RTE_SUBQUERY)
 					{
 						distributed = false;
+#if PG15_GE
+						Node *jtnode = (Node *) root->parse->jointree;
+						if (IsA(jtnode, FromExpr))
```

Comment:
Can we put this code for traversing the jointree hierarchy into the `distributed_rtes_wakler`?

Reply:
Not possible, since i dont have all context to parse jointree in distributed_rtes_wakler().